web-components/collapse/collapse.svelte

```diff
@@ -0,0 +1,131 @@
+<svelte:options tag="leo-collapse"/>
+
+<script lang="ts">
+import { createEventDispatcher } from "svelte";
+
+
+  export let title = ''
+  export let open: boolean | undefined
```

Comment:
nit: reduce to 1 line so we can easily see default value for attribute:
```
export let open: boolean | undefined = false
```

What do we think about naming boolean variables `isXYZ`. Usually I'd suggest that for React, but since this is more "native html-like", I'm unsure. But still, svelte does accept complex data for things, so sometimes it's difficult to tell if an attribute expects data or a simple boolean. In React, the main reason is that `open` could be a function to handle open instead of a directive (or reflection) to be open.

Either way, I'd like to be consistent accross components so we should change this or the button component and move forward with that.

@fallaciousreasoning @AlanBreck @nullhook

Reply:
I personally like the `isXYZ` naming convention.

Reply:
I've updated the name of the prop to be `isOpen`. The reason the props are split like this is so we can support two different scenarios:
1) Uncontrolled: The component Just Works when you drop it in, it can be expanded and collapsed.
2) Controlled: The owner of the component passes in `isOpen` and the component fires an event when it thinks it should be expanded/collapsed. An example of when this is useful might be when you want to be able to toggle the collapse with an external button.

Maybe there's a more Sveltey way of doing this, but this way ensures we get nice interop with React.

I've renamed the internal variable to be `isOpenInternal` to make this more obvious (we need to know that `isOpen` is undefined in order to control the collapse).